internal/common/common.go

```diff
@@ -190,3 +193,51 @@ func IsSliceNonEmpty(arg interface{}) bool {
 	}
 	return res
 }
+
+func getUnifiedNTPSources(db *gorm.DB, clusterID strfmt.UUID) (string, error) {
+	var sources []string
+	err := db.Raw("select distinct(additional_ntp_sources) as sources from infra_envs where id in (select distinct(infra_env_id) from hosts where cluster_id = ?)", clusterID.String()).Pluck("sources", &sources).Error
```

Comment:
Can you elaborate on why this query is here rather than in `internal/common/db.go`?

All the queries that access the database are in that file, having the query here will make it harder to refactor this code in the future.